### ISSUE TITLE
NettyConnectorProvider (jersey-netty-connector) doesn't send query parameters in the Get Request

### ISSUE THREAD

**Issue by luanjr** [eclipse-ee4j/jersey#4369]
I am trying to send an HTTP request with query parameters using Jersey Client and setting NettyConnectorProvider (org.glassfish.jersey.connectors:jersey-netty-connector:2.25.1) as a provider to the config (org.glassfish.jersey.client.ClientConfig) by the method connectorProvider().
I tried to send the query parameters using the method queryparam(), but the values are not sent on the request.

Looking in the source code of NettyConectorProvider, it uses the method requestUri.getRawPath()) during the creation of DefaultFullHttpRequest, but this method returns the path without the query parameters.

nettyRequest = new DefaultFullHttpRequest(HttpVersion.HTTP_1_1,
                                          HttpMethod.valueOf(jerseyRequest.getMethod()),
                                          requestUri.getRawPath());

How can I send query parameters in a GET request using NettyConnectorProvider or is there a knowledge issue related to query parameters?

**Comment by senivam** [comment]
Did you try:
```
requestUri.toASCIIString();
```
or 
```
requestUri.getRawPath() + "?" + requestUri.getRawQuery()
```
instead of 
```
requestUri.getRawPath()
```
as DefaultFullHttpRequest constructor parameter?

**Comment by jansupol** [comment]
Fixed by #4393
